src/odin/commands/status.rs

```diff
@@ -0,0 +1,49 @@
+use crate::server::ServerInfo;
+use crate::utils::environment::fetch_var;
+use crate::utils::parse_arg_variable;
+use clap::ArgMatches;
+use log::info;
+use std::env;
+
+pub fn fetch_public_address() -> String {
+  let current_ip = env::var("ADDRESS").unwrap_or_else(|_| {
+    reqwest::blocking::get("https://api.ipify.org")
+      .unwrap()
+      .text()
+      .unwrap()
+  });
+  let current_port: u16 = fetch_var("PORT", "2456").parse().unwrap();
+  format!("{:?}:{:?}", current_ip, current_port + 1)
```

Comment:
Wouldn't debug formatting on `current_ip` cause extra double quotes? I don't think debug formatting is really needed for either of these

Reply:
Yee my b